src/prerender_content.js

```diff
@@ -1,11 +1,15 @@
 /* eslint no-console: "off" */
-import "scrivito";
-import filesize from "filesize";
+/* eslint-disable-next-line import/no-extraneous-dependencies */
+import path from "path";
+import fse from "fs-extra";
 import "./Objs";
 import "./Widgets";
 import { configure } from "./config";
 import prerenderObjs from "./prerenderContent/prerenderObjs";
 import prerenderSitemap from "./prerenderContent/prerenderSitemap";
+import { extendRedirects } from "./prerenderContent/extendRedirects";
+import { reportError } from "./prerenderContent/reportError";
+import { storeResult } from "./prerenderContent/storeResult";
```

Comment:
Since `storeResult` is now just an import, we no longer need to pass it as a callback to `prerenderSitemap` or `prerenderObjs`. They can simply import `prerenderContent/storeResult` directly.